src/db/db.py

```diff
@@ -1,3 +1,3 @@
 from flask_sqlalchemy import SQLAlchemy
 
-database = SQLAlchemy()
\ No newline at end of file
+db = SQLAlchemy()
```

Comment:
This is all right, It think, as it is a common abbreviation and is less clutter.